plugins/modules/system/sudoers.py

```diff
@@ -63,20 +63,24 @@
 
 EXAMPLES = '''
 - name: Allow the backup user to sudo /usr/local/bin/backup
-  sudoers:
+  community.general.sudoers:
     name: allow-backup
     state: present
     user: backup
     commands: /usr/local/bin/backup
 
-- name: Allow the monitoring group to run sudo /usr/local/bin/gather-app-metrics without requiring a password
-  sudoers:
+- name: |
+    Allow the monitoring group to run sudo /usr/local/bin/gather-app-metrics
+    without requiring a password
```

Comment:
Maybe `>` instead of `|`? The pipe symbol will join the next indented lines but inserting a newline character betweem each line and the next. The greater-than symbol will simply concatenate (TBF, it will add a newline at the very end - to prevent that use `>-`).

Reply:
Wouldn't that be the same as having it on one long line - at least for the purposes of converting it into HTML?
https://github.com/ansible-collections/community.general/pull/3746#discussion_r757250857

Happy to go for the `>-` method if it improves the module.

Reply:
The difference is that the examples are rendered as-is without the YAML being interpreted first, so with `>-` you will have shorter lines in the YAML source, but for Ansible (when interpreting the playbook/task) it ends up as a long line. So it will avoid horizontal scrolling in the HTML docs, but still generate the same (long) task name in Ansible.

Reply:
I now see the distinction. Updated.